physionet-django/console/forms.py

```diff
@@ -449,23 +450,24 @@ class Meta:
     def __init__(self, responder, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.responder = responder
-        self.fields['status'].choices = CredentialApplication.REJECT_ACCEPT_WITHDRAW[:3]
+        self.fields['status'].choices = CredentialApplication.Status.choices_process_application
 
     def clean(self):
         if self.errors:
             return
 
-        if self.cleaned_data['status'] == 1 and not self.cleaned_data['responder_comments']:
+        if self.cleaned_data['status'] == CredentialApplication.Status.REJECTED\
```

Comment:
As above: There should be a space before the `\` I think, but PEP8 discourages `\` for wrapping long lines, suggesting parentheses instead: https://peps.python.org/pep-0008/#multiline-if-statements